SQL/database_schema.sql

```diff
@@ -1,8 +1,6 @@
 /**
  * make sure to bump schema version and mark changes in database_changelog.md!
  *
- * default prefix is rp_
```

Comment:
the reason this is prefixed is not because it's meant to be used as such
rename the file to `database_schema-template.sql` or something
the reason this file is here is so generating the actual `.sql`'s is as easy as find-replacing `%_PREFIX_%` instead of having to edit two files simultaneously. this results in a lot less typo's.